.clabot

```diff
@@ -101,6 +101,7 @@
     "abstractOwl",
     "milan-sumo",
     "parthiv-sumo"
+    "hganapathy"
```

Comment:
please add comma after "  "parthiv-sumo", this should resolve the problem https://github.com/SumoLogic/sumologic-documentation/pull/1859

Reply:
Oh! I missed it. Thank you :)